Indian/test.txt

```diff
@@ -0,0 +1 @@
+This final exercise on Udacity. Don't know what the story is about.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.